apps/web/components/EnterprisePage.tsx

```diff
@@ -42,7 +42,7 @@ export default function EnterprisePage() {
   ];
   return (
     <div>
-      <ShellMain heading={t("enterprise")} subtitle={t("enterprise_description")}>
+      <Shell heading={t("enterprise")} subtitle={t("enterprise_description")}>
```

Comment:
EnterprisePage was basically doing

<Shell withoutMain>
    <ShellMain>
        <children>
        
When it just could do

<Shell>
    <children>